### PR TITLE
blog re: patchfix to update sites for Mars.2...

### PR DESCRIPTION
**Pull request by nickboldt**
blog re: patchfix to update sites for Mars.2 (Egit 4.1.1)

### CODE REVIEW

File: blog/ga-patchfix-mars.2.adoc [hunk 1]
```diff
@@ -0,0 +1,39 @@
+= Patch fix for Red Hat JBoss Developer Studio 9.0.0.GA and JBoss Tools 4.3.0.Final
```
Comment (maxandersen) [inline blog/ga-patchfix-mars.2.adoc]: Patch fix is a political loaded word - and the title is a mile long ;)

Suggestion:
Fix for Developer Studio and Tools updatesites for Eclipse Mars.2

File: blog/ga-patchfix-mars.2.adoc [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+Updates to link:/downloads/devstudio/mars/9.0.0.GA.html[Red Hat JBoss Developer Studio 9] and link:/downloads/jbosstools/mars/4.3.0.Final.html[JBoss Tools 4.3.0.Final] for Eclipse Mars are now generally available!
+
+image::/blog/images/jbds9.png[]
+
+== What's new?
+
+We've added a link in the composite updates sites to EGit 4.1.1, so that users of Mars.2 can continue to enjoy using EGit with JBoss Tools and Red Hat JBoss Developer Studio.
+
+Without this minor change, installation of JBoss Tools and Red Hat JBoss Developer Studio on Mars.2 cause two versions of jgit to be installed, effectively disabling it and preventing EGit views and preferences from working.
```
Comment (maxandersen) [inline blog/ga-patchfix-mars.2.adoc]: add link to https://issues.jboss.org/browse/JBIDE-21821 ?

File: blog/ga-patchfix-mars.2.adoc [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+* link:/downloads/devstudio/mars/9.1.0.Beta2.html[9.1.0.Beta2]
+
+== Previous announcements
+
+For what's new in this release of JBoss Tools 4.3.0.Final or JBoss Developer Studio 9.0.0.GA, see our link:ga-for-mars.html[previous GA blog announcement].
+
+
+== What's next?
+
+We are already working on the next maintenance release for Eclipse Mars - JBoss Tools 4.3.1.CR1 and Red Hat JBoss Developer Studio 9.1.0.CR1 will be available shortly, with Final?GA releases soon after.
```
Comment (alexeykazakov) [inline blog/ga-patchfix-mars.2.adoc]: Final?GA -> Final/GA

File: blog/ga-patchfix-mars.2.adoc (hunk 2 above)
Comment (nickboldt) [inline blog/ga-patchfix-mars.2.adoc]: link is one line below.

File: blog/ga-patchfix-mars.2.adoc (hunk 1 above)
Comment (nickboldt) [inline blog/ga-patchfix-mars.2.adoc]: That's invalid according to the branding rules.

How about "EGit fix for Red Hat JBoss Developer Studio 9.0.0.GA and JBoss Tools 4.3.0.Final update sites" ?

**Pull request merged**